uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-month.component.spec.js

```diff
@@ -142,18 +152,22 @@
               }));
             });
 
+            it('loads leave requests for *enabled* absence types only', function () {
```

Comment:
it should be "for `*active*`"

Reply:
Not really, it would be inconsistent with what we have in DB, API and the interface:

![image](https://user-images.githubusercontent.com/3973243/32223330-d321cd70-be34-11e7-8e8e-12b744e476bc.png)

Reply:
The API accepts `is_active` parameter, so already it is not consistent as we can see. Its fine, please ignore.